lib/view_models/user_view_model.dart

```diff
@@ -9,10 +9,11 @@ import 'package:gocast_mobile/base/networking/api/handler/token_handler.dart';
 import 'package:gocast_mobile/base/networking/api/handler/user_handler.dart';
 import 'package:gocast_mobile/models/error/error_model.dart';
 import 'package:gocast_mobile/models/user/user_state_model.dart';
-import 'package:gocast_mobile/utils/globals.dart';
 import 'package:logger/logger.dart';
 import 'package:gocast_mobile/base/networking/api/handler/notification_handler.dart';
 
+import '../utils/globals.dart';
```

Comment:
It looks good to me. One thing, maybe you should change this line to`import 'package:gocast_mobile/utils/globals.dart';` for the consistency

Reply:
As far I know Package imports are preferred when accessing files across different packages. I think i would change all others in another PR. :)
We will have consistency

Reply:
If it will be fixed in another PR, it's ok for me